easybuild/scripts/bootstrap_eb.py

```diff
@@ -50,8 +51,7 @@
 from distutils.version import LooseVersion
 
 # set print_debug to True for detailed progress info
-print_debug = False
-#print_debug = True
+print_debug = os.environ.get('EASYBUILD_BOOTSTRAP_DEBUG', False)
```

Comment:
hmm, no, it's not, since the `0` is taken as a string value, and a non-empty string is equivalent to `True`

not sure how to tackle this one...

but since this is disabled by default, it's probably not a disaster if setting this to `0` enables it?